### PR TITLE
Translate to Portuguese: examples/mp-jwt-bean-validation

### PR DESCRIPTION
**Pull request by Daniel-Dos**
link to jira: https://issues.apache.org/jira/browse/TOMEE-2831

### CODE REVIEW

File: examples/mp-jwt-bean-validation/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,48 @@
+:index-group: MicroProfile
+:jbake-type: page
+:jbake-status: published
+
+= MicroProfile JWT com validação de bean
+
+TomEE tem uma caracteristica particular que permite o uso de Validação de Bean como alternativa ao complemento da anotação `@RolesAllowed`.
+
+A motivação para esta função é que `JsonWebToken` é efetivamente um bean, e uma verificação de seguraça é, em última instancia, um ato de produzir um resultado booleano utilizando uma entrada minima. È um problema perfeito para a validação de Bean.
```
Comment (hbelmiro) [inline examples/mp-jwt-bean-validation/README_pt.adoc:9]:
```suggestion
A motivação para esta função é que `JsonWebToken` é efetivamente um bean, e uma verificação de seguraça é, em última instância, um ato de produzir um resultado booleano utilizando uma entrada mínima. É um problema perfeito para a validação de Bean.
```

File: examples/mp-jwt-bean-validation/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+A caracteristica em última instancia lhe permite implementar um método como este:
```
Comment (hbelmiro) [inline examples/mp-jwt-bean-validation/README_pt.adoc:11]:
```suggestion
A funcionalidade lhe permite implementar um método como este:
```

File: examples/mp-jwt-bean-validation/README_pt.adoc (hunk 1 above)
Comment (Daniel-Dos) [reply examples/mp-jwt-bean-validation/README_pt.adoc]: done thanks

File: examples/mp-jwt-bean-validation/README_pt.adoc (hunk 2 above)
Comment (Daniel-Dos) [reply examples/mp-jwt-bean-validation/README_pt.adoc]: done thanks

**Comment by Daniel-Dos** [comment]
thanks @hbelmiro  for review .

**Pull request merged**